.github/workflows/cli.yaml

```
@@ -0,0 +1,92 @@
+name: Karmada CLI Workflow
```

```suggestion
name: CLI
```
We can use a short name